### PR TITLE
Translated part of non-translated polish strings

### PR DESCRIPTION
**Pull request by szymex73**

### CODE REVIEW

File: languages/pl.json [hunk 1]
```diff
@@ -91,6 +91,18 @@
   "workshopper-lololodash": "Naucz się, jak uprościć operacje na tablicach i obiektach za pomocą <em>Lo-Dash</em> (rozszerzona wersja biblioteki <code>underscore</code>).",
   "workshopper-learnyoucouchdb": "Dowiedz się więcej o <em>CouchDB</em> - bazie danych w pełni przygotowanej do obsługi aplikacji internetowych.",
   "workshopper-learnuv": "Naucz się <code>uv</code> aby dowiedzieć się co w Node.js <em>piszczy</em>. Przewodnik po bibliotece, która napędza Node.js.",
+  "workshopper-how-to-npm": "Naucz się jak używać i tworzyć moduły npm.",
+	"workshopper-elementary-electron": "Stwórz desktopową aplikację używając Node i Chromium poprzez Electron.",
```
Comment (riyadhalnur) [inline languages/pl.json]: Formatting is off from L95-L105. Otherwise, LGTM

Comment (riyadhalnur) [inline languages/pl.json]: Formatting is off from L95-L105. Otherwise, LGTM

**Comment by szymex73** [comment]
Fixed formatting on L95-L105 as requested

**Comment by riyadhalnur** [comment]
Thank you! Merged 🚀

**Pull request merged**